plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -1043,13 +1044,15 @@ public void testIoExplain()
         computeActual("CREATE TABLE test_io_explain WITH (partitioned_by = ARRAY['orderkey', 'processing']) AS SELECT custkey, orderkey, orderstatus = 'P' processing FROM orders WHERE orderkey < 3");
 
         EstimatedStatsAndCost estimate = new EstimatedStatsAndCost(2.0, 40.0, 40.0, 0.0, 0.0);
+        boolean withoutData = estimate.getOutputRowCount() == 0d ? true : false;
         MaterializedResult result = computeActual("EXPLAIN (TYPE IO, FORMAT JSON) INSERT INTO test_io_explain SELECT custkey, orderkey, processing FROM test_io_explain WHERE custkey <= 10");
         assertEquals(
                 getIoPlanCodec().fromJson((String) getOnlyElement(result.getOnlyColumnAsSet())),
                 new IoPlan(
                         ImmutableSet.of(
                                 new TableColumnInfo(
                                         new CatalogSchemaTableName(catalog, "tpch", "test_io_explain"),
+                                        new IoPlanPrinter.Constraint(withoutData,
                                         ImmutableSet.of(
```

Comment:
the indentation here is off and will change as soon as someone invokes reformatting

```suggestion
                                        new IoPlanPrinter.Constraint(withoutData, ImmutableSet.of(
```